tests/test_cln_rs.py

```diff
@@ -178,11 +179,16 @@ def test_grpc_generate_certificate(node_factory):
 
 def test_grpc_no_auto_start(node_factory):
     """Ensure that we do not start cln-grpc unless a port is configured.
+    Also check that we do not generate certificates.
     """
     l1 = node_factory.get_node()
 
     wait_for(lambda: [p for p in l1.rpc.plugin('list')['plugins'] if 'cln-grpc' in p['name']] == [])
     assert l1.daemon.is_in_log(r'plugin-cln-grpc: Killing plugin: disabled itself at init')
+    p = Path(l1.daemon.lightning_dir) / TEST_NETWORK
+    files = os.listdir(p)
+    pem_files = [f for f in files if re.match(r".*\.pem$",f)]
```

Comment:
```suggestion
    pem_files = [f for f in files if re.match(r".*\.pem$", f)]
```

This cause the following error

```
2023-10-10T09:08:00.8610981Z checking for ANSI C header files... yes
2023-10-10T09:08:00.9901583Z checking for sys/types.h... yes
2023-10-10T09:08:01.1104640Z checking for sys/stat.h... tests/test_cln_rs.py:190:58: E231 missing whitespace after ','
2023-10-10T09:08:01.1687731Z yes
2023-10-10T09:08:01.1774097Z make: *** [Makefile:538: check-python-flake8] Error 1
2023-10-10T09:08:01.1774740Z make: *** Waiting for unfinished jobs....
2023-10-10T09:08:01.3928110Z checking for stdlib.h... yes
2023-10-10T09:08:01.6206722Z checking for string.h... yes
```